content/hi/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md

```diff
@@ -0,0 +1,68 @@
+जिसके वैल्यू आपके माइक्रोसर्विसेज में इंजेक्ट किए जाएंगे। कॉन्फिगमैप्स का उपयोग करने के लाभों में से एक और
+रहस्य यह है कि इन्हें असाइन किए जाने सहित कई कंटेनरों में पुन: उपयोग किया जा सकता है
+विभिन्न कंटेनरों के लिए अलग-अलग एनवायरनमेंट वेरिएबल।
+
+कॉन्फिगमैप्स एपीआई ऑब्जेक्ट हैं जो गैर-गोपनीय कीय-वैल्यू पेयर को स्टोर करते हैं। इंटरैक्टिव में
+ट्यूटोरियल में आप सीखेंगे कि एप्लिकेशन के नाम को स्टोर करने के लिए कॉन्फिग मैप का उपयोग कैसे करें। अधिक जानकारी के लिए
+कॉन्फ़िगमैप्स के संबंध में जानकारी, आप दस्तावेज़ पा सकते हैं
+[यहां](/docs/tasks/configure-pod-container/configure-pod-configmap/)।
+
+हालाँकि सीक्रेट्स का उपयोग कीय-वैल्यू पेयर को स्टोर करने के लिए भी किया जाता है, लेकिन वे कॉन्फ़िगमैप्स से भिन्न होते हैं
+वे गोपनीय/संवेदनशील जानकारी के लिए अभिप्रेत हैं और बेस64 एन्कोडिंग का उपयोग करके स्टोर किए जाते हैं।
+यह रहस्यों को क्रेडेंशियल्स, कीय आदि जैसी चीज़ों को स्टोर करने के लिए जिसका उपयुक्त विकल्प बनाता है
+टोकन, जिनमें से पहला आप इंटरैक्टिव ट्यूटोरियल में करेंगे। अधिक जानकारी के लिए
+रहस्य, आप दस्तावेज़ [यहां](/docs/concepts/configuration/secret/) पा सकते हैं।
+
+
+### कोड से कॉन्फ़िग को इक्स्टर्नलाइज़ बनाना {# externalizing-config-from-code}
+
+बाहरी एप्लिकेशन कॉन्फ़िगरेशन उपयोगी है क्योंकि कॉन्फ़िगरेशन आमतौर पर इसके आधार पर बदलता रहता है
+आपके एनवायरनमेंट पर इसे पूरा करने के लिए, हम जावा के संदर्भ और निर्भरता का उपयोग करेंगे
+इंजेक्शन (सीडीआई) और माइक्रोप्रोफाइल कॉन्फ़िगरेशन। माइक्रोप्रोफाइल कॉन्फिग माइक्रोप्रोफाइल की एक विशेषता, एक सेट है
+क्लाउड-नेटिव माइक्रोसर्विसेज के विकास और तैनाती के लिए खुली जावा प्रौद्योगिकियों का।
+
+सीडीआई एक 
+वैल्यू निर्भरता इंजेक्शन क्षमता प्रदान करता है जो किसी एप्लिकेशन को असेंबल करने में सक्षम बनाता है
+सहयोग करने से, शिथिल-युग्मित फलियाँ। माइक्रोप्रोफाइल कॉन्फिग ऐप्स और माइक्रोसर्विसेज प्रदान करता है
+एप्लिकेशन, रनटाइम सहित विभिन्न स्रोतों से कॉन्फ़िगरेशन गुण प्राप्त करने का 
+वैल्यू तरीका
+और एनवायरनमेंट. स्रोत की परिभाषित प्राथमिकता के आधार पर, गुण स्वचालित रूप से होते हैं
+गुणों के एक सेट में संयोजित किया गया है जिसे एप्लिकेशन एपीआई के माध्यम से एक्सेस कर सकता है। एक साथ,
+बाहरी रूप से प्रदान की गई सामग्री को पुनः प्राप्त करने के लिए इंटरैक्टिव ट्यूटोरियल में सीडीआई और माइक्रोप्रोफाइल का उपयोग किया जाएगा
+कुबेरनेट्स कॉन्फिगमैप्स और सीक्रेट्स से गुण और आपके एप्लिकेशन कोड में इंजेक्ट हो जाते हैं।
+
+कई ओपन सोर्स फ्रेमवर्क और रनटाइम माइक्रोप्रोफाइल कॉन्फ़िगरेशन को लागू और समर्थन करते हैं। लगातार
+इंटरैक्टिव ट्यूटोरियल में, आप ओपन लिबर्टी का उपयोग करेंगे, जो एक लचीला ओपन-सोर्स जावा रनटाइम है
+क्लाउड-नेटिव ऐप्स और माइक्रोसर्विसेज बनाना और चलाना। हालाँकि, कोई भी माइक्रोप्रोफाइल संगत है
+इसके बजाय रनटाइम का उपयोग किया जा सकता है।
+
+
+## {{% heading "objectives" %}}
+
+* एक कुबेरनेट्स कॉन्फ़िगमैप और सीक्रेट बनाएं
+* माइक्रोप्रोफाइल कॉन्फ़िगरेशन का उपयोग करके माइक्रोसर्विस कॉन्फ़िगरेशन इंजेक्ट करें
+
+<!-- lessoncontent -->
+
+## उदाहरण: माइक्रोप्रोफाइल, कॉन्फिगमैप्स और सीक्रेट्स का उपयोग करके कॉन्फिगरेशन को इक्स्टर्नलाइज़ बनाना
```

Comment:
Perhaps it would be beneficial to utilize English text for this hyperlink to avoid any non-English characters appearing in the URL of the section.

Reply:
I have made the changes please check it